web/l10n/en.json

```diff
@@ -118,6 +118,7 @@
     "attributeUiDisableCalendars": "UI: Disable Calendars",
     "attributeUiDisableMaintenance": "UI: Disable Maintenance",
     "attributeUiHidePositionAttributes": "UI: Hide Position Attributes",
+    "attributeUiDisableStylesAttributes": "UI: Disable Styles",
```

Comment:
The key should be renamed to something like `attributeUiDisableMapLayers`. It definitely shouldn't have `Attributes` in the name. And also "styles" won't be as clear for people because it's a technical term specifically used by this map library. It's not a common term people would understand.

And the value should be "UI: Disable Map Layers".

Reply:
Agreed. I don't know why I double added the word attribute(s). Also I changed the word styles to layers on the attributes side; within the useMapStyles script it still uses the word styles to align with the script. 

Changed was made in [22ef038](https://github.com/traccar/traccar-web/pull/970/commits/22ef038f139f0bc6be7f97f792f0988356593204)